pkgs/top-level/ocaml-packages.nix

```diff
@@ -1701,7 +1701,9 @@ in let inherit (pkgs) callPackage; in rec
 
   ocamlPackages_5_0 = mkOcamlPackages (callPackage ../development/compilers/ocaml/5.0.nix { });
 
-  ocamlPackages_latest = ocamlPackages_5_0;
+  ocamlPackages_5 = ocamlPackages_5_0;
```

Comment:
```suggestion
  ocamlPackages_4 = ocamlPackages_4_14;
  ocamlPackages_5 = ocamlPackages_5_0;
```

Reply:
What should `ocamlPackages_5` mean when `ocaml-ng.ocamlPackages` and `ocaml-ng.ocamlPackages_latest` both reach 5.0?

Reply:
> What should `ocamlPackages_5` mean when `ocaml-ng.ocamlPackages` and `ocaml-ng.ocamlPackages_latest` both reach 5.0?

It seems: `ocaml-ng.ocamlPackages` is the blessed version. And latest being the last available. I also don't like the `_latest` much. But, for now makes sense.

`ocamlPackages_5` if fixed and will keep existing until removed.

Reply:
As I got to think more of it, latest does not seem necessary. What actually is needed is to bump default to latest. Phew...